### ISSUE TITLE
Multiple test files?

### ISSUE THREAD

**Issue by shadowspawn** [pkgjs/parseargs#66]
Light question. Does the current testing infrastructure using `tape` and `c8` support multiple test files, without writing a test runner? e.g. adding `test/multiple.js`

---

I have not previously used `tape` or `c8`. After some research and experimentation I found this appears to work on Mac, but I am not sure about Windows. (Or whether it is actually doing what I intend! 🤔).

```
npx c8 tape test/*.js
```

**Comment by ljharb** [comment]
Yes, tape supports multiple files - altho paths with asterisks need to be quoted to prevent the shell interpreting them first.
